azkaban-common/src/main/java/azkaban/utils/TypedMapWrapper.java

```diff
@@ -91,6 +91,24 @@ public Long getLong(final K key, final Long defaultVal) {
     }
   }
 
+  public Double getDouble(final K key) {
```

Comment:
I studied this class `TypeMapWrapper`, and it is used for metadata wrapping. Java Generics is not necessary, as AZ always initialize a <String, Object> instance. We may refactor it in future.